admin/qtx_configuration.php

```diff
@@ -1392,7 +1392,7 @@ function qtranxf_conf() {
 					</fieldset><br />
 					<textarea id="highlight_mode_custom_css" name="highlight_mode_custom_css" style="width:100%"><?php echo esc_attr($highlight_mode_custom_css); ?></textarea>
 					<br />
-					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme').'</a>"') ?></small>
+					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme', 'qtranslate').'</a>"') ?></small>
```

Comment:
I saw that this was already on the main WordPress .po because it's a link that appeared translated within a sentence not translated. So, I've found it's origin and translated it like it is in the main .po.
Yes, I think that the .po must be 100%, of course if you use a lot of strings similar to main WordPress localization files qTranslate will appear at leas somehow translated, but in the end that is not what's intended.

In the end I think it's not worthy to you to try to use strings already translated in the main WordPress, to brain expensive :)
With or without textdomain, if a specific localization don't exist, qTranslate will get it from the main files, so there is no difference, please always include it.

Translators should of course make their work perfectly according to WordPress terms, rules and guidelines.
This reminds me of the Translation Guide that we've not completed yet :)